documentation/library_guide/kernel_templates/esimd/radix_sort.rst

```diff
@@ -1,37 +1,37 @@
 Radix Sort
 ##########
 
----------------------------------------------------
-radix_sort and radix_sort_by_key Function Templates
----------------------------------------------------
+-----------------------------
+radix_sort Function Templates
+-----------------------------
 
-The ``radix_sort`` and ``radix_sort_by_key`` functions sort data using the radix sort algorithm.
+The ``radix_sort`` function sorts data using the radix sort algorithm.
 The sorting is stable, ensuring the preservation of the relative order of elements with equal keys.
 The functions implement a Onesweep* [#fnote1]_ algorithm variant. Both in-place and out-of-place
 overloads are provided. For out-of-place overloads, the input data order is preserved.
 
-A synopsis of the ``radix_sort`` and ``radix_sort_by_key`` functions is provided below:
+A synopsis of the ``radix_sort`` function is provided below:
 
 .. code:: cpp
 
    // defined in <oneapi/dpl/experimental/kernel_templates>
 
    namespace oneapi::dpl::experimental::kt::gpu::esimd {
 
-   // Sort a single sequence
+   // Sort in-place
    template <bool IsAscending = true, std::uint8_t RadixBits = 8,
              typename KernelParam, typename Iterator>
    sycl::event
    radix_sort (sycl::queue q, Iterator first, Iterator last,
-               KernelParam param); // (1)
+               KernelParam param) // (1)
```

Comment:
I think it is better to add missing semicolons than remove proper ones. Essentially, the synopsis consists of forward declarations, and the closing semicolon is required for those.

Reply:
Added